lms/static/sass/views/_program-details.scss

```diff
@@ -630,10 +584,18 @@
   padding: 30px 10px;
 
   .program-record {
-    text-align: center;
+    text-align: left;
     padding-bottom: 2em;
   }
 
+  .view-record-wrapper {
+    text-align: right;
+
+    .view-record-button {
+      padding: 7px 18.5px 7px 18.5px;
```

Comment:
it would be great to use padding variables here or multiples of $baseline (20px). We might even have updated patterns / standards here that I'm not aware of. Avoiding things like 7px or 18.5px would be good.

Reply:
I agree, it's definitely not ideal, just the standard for the rest of the program-details.scss. Do we want to do a broader conversion to using multiples of $baseline across the file?